### DISCUSSION TITLE
v0.2.0

### DISCUSSION THREAD

**Discussion by hensm** [hensm/fx_cast#208] [General]
## Known issues
* Local media casting broken on some platforms.
* No macOS x86 build or Windows 32-bit x86 build due to cross compiling issues that I don't have time to figure out (contributions welcome). Build instructions for the app can be found in the project README.
* Possibly more, haven't done any extensive testing.

## Changelog

* Improved cast API compatibility:
  * Added Queue support (commit: cff7466)
  * Added newer cast API additions (e4e133c)
  * Fixed `cast.media.Media#getEstimatedTime` (commit: c117241)
  * Fixed media update listener handling (commit: 119d6c8)
  * Fixed device capability handling (commit: b672b8d)
  * Fixed supported media commands handling (commit: 61f9ff0)
  * Fixed issue with `window.cast` property causing errors on pages using CAF. #205 
* Added button in receiver selector UI to suggest whitelisting known sites (PR: #204).
  ![](https://user-images.githubusercontent.com/6956633/163699688-ffddac4b-a5d0-4d56-9b8a-e628c0e9cf71.png)
* Added image support to media sender. #183
* Added Norwegian localization (@jorgenboganes). (commit: fcce716)
* Update Spanish localizations.
* Update extension UI style to match updated Firefox style + misc style changes.
* Removed native macOS receiver selector.
* Removed experimental media overlay (commit: 59dc806)

<hr /><em>This discussion was created from the release <a href='https://github.com/hensm/fx_cast/releases/tag/v0.2.0'>v0.2.0</a>.</em>

**Comment by lucianocrimi** [comment]
Non funziona su Mac 10.14.6 Mojave e firefox 104.0

**Reply by hensm** [reply]
See known issues. Use `v0.1.2` until next release.

**Reply by lucianocrimi** [reply]
it doesn't work too

**Reply by hensm** [reply]
Added an x86 `v0.2.0` macOS build.

**Reply by lucianocrimi** [reply]
Now i tested on youtube and it does work. There is a great problem. My chromecast remains connected with youtube. I tried to close browser but chromecast continues to transmit from youtube. I open another site (NETFLIX) but i cannot send anything.

**Reply by hensm** [reply]
Yeah, it's supposed to stay connected. It's a bit weird, but hold shift or option and you should get a close button in place of a cast button in the popup. The next version will have some major UX improvements in this area and be a lot more intuitive.

Not sure about your other issue, I can't reproduce it (also tried it with Netflix). Maybe open an issue with some more detail and [browser console](https://firefox-source-docs.mozilla.org/devtools-user/browser_console/index.html) logs if relevant.
